client/my-sites/comments/comment-list/index.jsx

```diff
@@ -38,6 +40,73 @@ export class CommentList extends Component {
 		}
 	}
 
+	deleteForever = commentId => () => {
+		this.props.removeNotice( `comment-notice-${ commentId }` );
+		this.showNotice( commentId, 'delete', 'trash' );
+
+		this.setState( { comments: omit( this.state.comments, commentId ) } );
+	}
+
+	getEmptyMessage = () => {
+		const { status, translate } = this.props;
+
+		switch ( status ) {
+			case 'unapproved':
+				return {
+					title: translate( 'No new comments yet.' ),
+					line: translate( 'Your queue is clear.' ),
+				};
+			case 'approved':
+				return {
+					title: translate( 'No approved comments.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **70** times:
`translate( 'No approved comments' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).